gemrb/core/DialogHandler.cpp

```diff
@@ -404,8 +403,10 @@ bool DialogHandler::DialogChoose(unsigned int choose)
 				EndDialog();
 				return false;
 			}
+
+			auto targetID = core->GetDictionary().GetAs("DLG_TARGET_ID", 0);
```

Comment:
Should be DLG_TARGET if anything. But since only DialogHandler is setting it, why do we need another lookup? `targetID` should already match the var.